### PR TITLE
Supplier product form integration and js

### PR DESCRIPTION
**Pull request by aleeks**
| Questions | Answers |
| --- | --- |
| Branch? | develop |
| Description? | Do supplier form on product page form like integration page. template & js files updated. |
| Type? | bug fix |
| Category? | BO |
| BC breaks? | no |
| Deprecations? | no |
| Fixed ticket? | http://forge.prestashop.com/browse/BOOM-996 |
| How to test? | Play with supplier form in product admin form |

#### Important guidelines
- Make sure [your local branch is up to date](https://help.github.com/articles/syncing-a-fork/) before commiting your changes!
- Your code MUST respect [our Coding Standards](http://doc.prestashop.com/display/PS16/Coding+Standards) (for code written in PHP, JavaScript, HTML/CSS/Smarty/Twig, SQL)!
- Your commit name MUST respect our [naming convention](http://doc.prestashop.com/display/PS16/How+to+write+a+commit+message)!

### CODE REVIEW

**Comment by vincentbz** [comment]
Tested and ok, thanks !

File: admin-dev/themes/default/js/bundle/product/form.js [hunk 1]
```diff
@@ -446,27 +446,29 @@ var featuresCollection = (function() {
  * Suppliers management
  */
 var supplier = (function() {
-  var defaultSupplierRow = $('#default_supplier_list');
-  var isInit = false;
+
+  var supplierInputManage = function(input) {
+    var supplierDefaultInput = $('#form_step6_suppliers input[name="form[step6][default_supplier]"][value=' + $(input).val() +']');
+    if($(input).is(':checked')) {
+      $(supplierDefaultInput).prop('disabled', false).show();
```
Comment (thierrymarianne) [inline admin-dev/themes/default/js/bundle/product/form.js]: `supplierDefaultInput` is a jQuery element so there is no need to wrap it again with $().

Comment (aleeks) [inline admin-dev/themes/default/js/bundle/product/form.js]: It's changed !

**Comment by thierrymarianne** [comment]
@aleeks Can you please rebase your branch against develop?

**Comment by aleeks** [comment]
@thierrymarianne rebased !

**Comment by thierrymarianne** [comment]
Thanks!

**Pull request merged**